### PR TITLE
use django's meta class for translatable fields

### PR DESCRIPTION
**Pull request by tatterdemalion**

### CODE REVIEW

File: tests/models.py [hunk 1]
```diff
@@ -9,7 +9,8 @@ class Fruit(TranslationModel):
     benefits = models.CharField(max_length=255)
     scientific_name = models.CharField(max_length=255)
 
-    translatable_fields = ['name', 'benefits']
+    class Meta:
+        translatable_fields = ['name', 'benefits']
```
Comment (bahattincinic) [inline tests/models.py]: Why did you use tuple instead of list ?

File: nece/models.py [hunk 2]
```diff
@@ -12,6 +13,8 @@
 else:
     from nece.fields.pgjson import JSONField
 
+options.DEFAULT_NAMES = options.DEFAULT_NAMES + ('translatable_fields',)
```
Comment (bahattincinic) [inline nece/models.py]: Maybe:

``` python
options.DEFAULT_NAMES += ('translatable_fields',)
```

**Comment by bahattincinic** [comment]
Can you please update documentation ? (https://github.com/tatterdemalion/django-nece/blob/master/README.rst#usage)

File: tests/models.py (hunk 1 above)
Comment (tatterdemalion) [inline tests/models.py]: What do you mean? It is a list.

File: nece/models.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from distutils.version import StrictVersion
 from django import get_version
 from django.db import models
+import django.db.models.options as options
```
Comment (erkanay) [inline nece/models.py]: why not `from django.db.models import options` ?

File: tests/models.py (hunk 1 above)
Comment (bahattincinic) [inline tests/models.py]:
``` python
class Meta:
    translatable_fields = ('name', 'benefits',)
```

**Comment by bahattincinic** [comment]
LGTM

**Pull request merged**